extensions/amp-story/1.0/analytics.js

```diff
@@ -28,6 +28,15 @@ const Events = {
   STORY_UNMUTED: 'story-audio-unmuted',
 };
 
+/** @enum {string} */
+export const AdvancementMode = {
+  GO_TO_PAGE: 'goToPageAction',
+  AUTO_ADVANCE_TIME: 'autoAdvanceTime',
+  AUTO_ADVANCE_MEDIA: 'autoAdvanceMedia',
+  MANUAL_ADVANCE: 'manualAdvance',
+  ADVANCE_TO: 'advanceTo',
+  ADVANCE_TO_ADS: 'advanceToAds',
```

Comment:
nit: this reads like it is saying the advancement was to an ad, where you're really saying the advancement was *from* an ad with the `advance-to` attribute.  Since publishers don't set the `advance-to` attribute for ads (it's set by us internally), I'm not sure the `advance-to` part is necessary to tell them, here.

Maybe we can rename this to something like `'manualAdvanceFromAd'`?

Reply:
Thanks for catching this! You're right that the variable name is misleading. 

Done.